scripts/reorder_json_keys.py

```diff
@@ -0,0 +1,32 @@
+import json
+import os
+
+from recipe_scrapers._utils import get_abstract_methods
+
+KEYS = get_abstract_methods()
+
+
+def reorder_json_keys(file_path):
+    with open(file_path, encoding="utf-8") as file:
+        data = json.load(file)
+
+    reordered_data = {key: data[key] for key in KEYS if key in data}
+
+    if list(data.keys()) != KEYS:
```

Comment:
Ah, sorry - I think I misunderstood part of this comparison before making my suggestion.  It's possible (in fact, quite likely) that the `data.keys()` won't contain the complete set of `KEYS`, isn't it?

So I think this should be:

```suggestion
    reordered_data = {key: data[key] for key in KEYS if key in data}

    if list(data.keys()) != list(reordered_data.keys()):
```